docs/setuptools.txt

```diff
@@ -970,7 +970,7 @@ to do things like this::
         packages=find_packages('src'),  # include all packages under src
         package_dir={'':'src'},   # tell distutils packages are under src
 
-        include_package_data=True,    # include everything in source control
+        include_package_data=True,  # include package data as per MANIFEST.in
```

Comment:
I should say also, `MANIFEST.in` isn't actually the source. `MANIFEST` is. It's just that most people rely on a MANIFEST.in template to generate MANIFEST.